clients/src/main/java/org/apache/kafka/clients/consumer/internals/Fetcher.java

```diff
@@ -703,15 +703,23 @@ public void recordTopicFetchMetrics(String topic, int bytes, int records) {
             // record bytes fetched
             String name = "topic." + topic + ".bytes-fetched";
             Sensor bytesFetched = this.metrics.getSensor(name);
-            if (bytesFetched == null)
+            if (bytesFetched == null) {
                 bytesFetched = this.metrics.sensor(name);
+                bytesFetched.add(this.metrics.metricName(name,
```

Comment:
We also have a "bytes-fetched" sensor which covers all topics and exposes the following metrics: "fetch-size-avg," "fetch-size-max," and "bytes-consumed-rate." Do you think we should have the same metrics at the topic level for consistency?